packages/frontend/src/pages/layer3s/props/getProjectDetails.ts

```diff
@@ -118,6 +120,14 @@ export function getProjectDetails(
       },
     })
 
+    const riskSection = getRiskSection(project, verificationStatus)
```

Comment:
lets put getters in the top of this function

Reply:
[6e2457b](https://github.com/l2beat/l2beat/pull/3360/commits/6e2457b0e61c48b19486b2abd9c46711dd18dfc0)